src/actions/markup.js

```diff
@@ -9,7 +9,6 @@ import { documentNameSelector } from '@/selectors/document'
 import {
   markupSelector,
   pageSelectedLabelsSelector,
-  pageSelectedAreasSelector,
```

Comment:
1. drop AREA_BORER from theme.default as never used
2. and rename AREA_BACKGROUND to MERGE_PROJECTION_BACKGROUND

Reply:
Done